spec/helpers.lua

```diff
@@ -2509,6 +2510,75 @@ local function build_go_plugins(path)
   end
 end
 
+local function isnewer(path_a, path_b)
+  if not pl_path.exists(path_a) then
+    return true
+  end
+  if not pl_path.exists(path_b) then
+    return false
+  end
+  return assert(pl_path.getmtime(path_b)) > assert(pl_path.getmtime(path_a))
+end
+
+local function make(workdir, specs)
```

Comment:
looks like we are replicating `make` here, should we instead use a `Makefile`?

Reply:
yes, i found writing this was easier than finding out if `make` is available in CI